shell/app/org-home/pages/personal-home.tsx

```diff
@@ -13,14 +13,18 @@
 
 import * as React from 'react';
 import DiceConfigPage from 'config-page/index';
-import routeInfoStore from 'app/common/stores/route';
 import orgStore from 'app/org-home/stores/org.tsx';
 import './personal-home.scss';
+import i18n from 'i18n';
 
 const PersonalHome = () => {
   const curOrgName = orgStore.useStore(s => s.currentOrg.name);
   const inParams = { orgName: curOrgName || '-' };
-  console.log(inParams)
+
+  React.useEffect(() => {
+    document.title = i18n.t('Personal dashboard') + ' · Erda';
+  }, []);
```

Comment:
Why not use route breadcrumb name?

Reply:
> Why not use route breadcrumb name?

because personal home page didn't use Header component